src/app.js

```diff
@@ -4,13 +4,20 @@ import cmsAdapterRecentScreenings from "./cmsAdapterRecentScreenings.js";
 import getRecentScreenings from "./getRecentScreenings.js";
 import { getMovie, getMovies, getMovieScreenings } from "./movies.js";
 import { marked } from "marked";
+import { builder } from "./buildReviewBody.js";
+import { parser as reviewParser } from "./postReviewParser.js";
```

Comment:
Here's an example of the "problem" with not using the "full" name of the function when the function is created. VSCode wants to import it as it's real name, but you want another name for it because the real name is too generic.